src/main/java/org/dimdev/riftloader/RiftLoader.java

```diff
@@ -127,7 +132,12 @@ private void findMods(File modsDir) {
                 log.error("Exception while checking if file " + file + " is a mod", t);
             }
         }
+    }
 
+    private void findMods() {
+        findModsOnClasspath();
+        findModsInDir(modsDir);
+        findModsInDir(new File(modsDir, "1.13"));
```

Comment:
Why not use `RealmsSharedConstants.VERSION_STRING`?

> Currently, `1.13` is also hardcoded in the `Main` class.

It is inlined by the compiler, isn't it?